tutorials/tutorial_workflows_v2.md

```diff
@@ -0,0 +1,102 @@
+Once the Workflows tool has been enabled and configured in an account, app developers are able to interact with the workflow objects to perform workflow activities via the Procore API.
+
+## Access
+
+Self-Serve Workflows endpoints are in closed beta. For inquiry and access, reach out to [apisupport@procore.com](mailto:apisupport@procore.com).
+
+---
+
+## Hierarchy of Workflows
+
+There are 3 main levels of a workflow: template, preset and instance.
+
+![API Record Diagram]({{ site.baseurl }}/assets/guides/workflows-v2-api-records-diagram.png)
+
+### Template
+
+A template is the container for a workflow configuration with one or many versions.
+It contains information that doesn't change from one version to the next (name, the related Procore tool, etc).
+
+A version is one configuration of a template. Either draft, active (latest published) or inactive (older published).
+It contains the configurations for steps, conditionals, responsible groups, transitions, etc.
+
+### Preset
+
+A preset is another layer of configuration on top of the latest published version.
+A company-level Procore tool will have one preset for the company and a project-level tool can have any number of presets based on the number of projects assigned to the template.
+The latest published version configuration and a preset or instance configuration (which gets copied from the preset when created) come together to create the full configuration (shown in the preset UI).
+
+When an instance is started, it copies the current configuration of the preset specified (or the default if none is specified).
+
+### Instance
+
+An instance is a workflow running for a Procore tool item (Invoice, Prime Contract, etc).
+Each step of an instance will create one step occurrence and however many response occurrences for the step's assignees.
+
+---
+
+## Things to Consider
+
+Before starting out with our Workflows API, there are a few things to consider.
+
+### Company and Project-Level Tools
+
+Using the [list template tool configurations API](https://developers.procore.com/reference/rest/v1/template-tool-configurations?version=1.0), a tool's "provider" can be determined as "Project" or "Company".
+This is how to determine which API endpoints should be used for the specific tool. Another way to tell, for the most part, is the context in which the tool is used when in Procore.
+
+### Update and Replace Fields
+
+There are a few fields where the field's value in the request will fully replace the existing field's value in our record.
+For example, for a preset's `responsible_group_memberships` array, you might expect a merge by only providing the group you want updated.
+That's not what will happen.
+You **must** provide the full shape of the original with any changes.
+
+Fields with this caveat will be noted.
```

Comment:
We have to make a few updates to our endpoint docs to get this right. Should just be something we have to note on some fields for the version and preset.